norfair/drawing.py

```diff
@@ -1,10 +1,18 @@
+from typing import Optional, Tuple, List
+
 import cv2
 import numpy as np
 
 from .utils import validate_points
 
 
-def draw_points(frame, detections, radius=None, thickness=None, color=None):
+def draw_points(
+    frame: np.array,
+    detections: List,
+    radius: Optional[int] = None,
+    thickness: Optional[int] = None,
+    color: Optional[Tuple] = None
```

Comment:
This could be a bit more specific by specifying that the Tuple should contain 3 ints.